src/libraries/Common/src/System/Net/Security/SslKeyLogger.cs

```diff
@@ -21,7 +21,7 @@ static SslKeyLogger()
 #if DEBUG
             bool isEnabled = true;
 #else
-            bool isEnabled = AppContext.TryGetSwitch("System.Net.Security.EnableSslKeyLogging", out bool enabled) && enabled;
+            bool isEnabled = AppContext.TryGetSwitch("System.Net.EnableSslKeyLogging", out bool enabled) && enabled;
```

Comment:
EnableTlsKeyLogging ???

Reply:
While technically more accurate, we use SSL in SSLKEYLOGFILE, let's not mix another abbreviation in.